src/components/icon/icon.ts

```diff
@@ -43,6 +41,8 @@ export default class IgcIconComponent extends SizableMixin(LitElement) {
    *
    * @attr [name=""]
    */
+  @property()
+  @alternateName('iconName')
```

Comment:
@rkaraivanov This seems to mess with the current function of `@alternateName`, seems to be read off of setters only.
// CC @gmurray81

Reply:
Actually, `  /* alternateName: iconName */` on the setter seems to also do the job, so we might default to that

Reply:
Simple prop works too :)